File: app/src/main/java/com/example/todoapp/DetailActivity.java
package com.example.todoapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class DetailActivity extends AppCompatActivity {
    TextView title;
    TextView description;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindView();
        setIntentData();
    }

    private void setIntentData() {
        Intent intent=getIntent();
        title.setText(intent.getStringExtra(AppConstant.TITLE));
        description.setText(intent.getStringExtra(AppConstant.DESCRIPTION));

    }

    private void bindView() {
        title=findViewById(R.id.TextViewTitle_Detail);
        description=findViewById(R.id.TextViewDescription_Detail);
    }
}

File: app/src/main/java/com/example/todoapp/DetailActivity.kt
package com.example.todoapp

import android.content.Intent
import android.os.Bundle
import android.util.Log
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class DetailActivity:AppCompatActivity() {
    lateinit var title: TextView
    lateinit var description: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindView()
        setIntentData()
    }

    private fun setIntentData() {

        val intent=intent
        title.text = intent.getStringExtra(AppConstant.TITLE)
        //Log.d("DetailActivity",intent.getStringExtra(AppConstant.TITLE))
        description.text = intent.getStringExtra(AppConstant.DESCRIPTION)
    }

    private fun bindView() {

        title=findViewById(R.id.TextViewTitle_Detail)
        description=findViewById(R.id.TextViewDescription_Detail)
    }
}